syft/frameworks/torch/hook/hook.py

```diff
@@ -601,8 +617,12 @@ def module_send_(nn_self, *dest, force_send=False, **kwargs):
             if module_is_missing_grad(nn_self):
                 create_grad_objects(nn_self)
 
-            for p in nn_self.parameters():
-                p.send_(*dest, **kwargs)
+            # for p in nn_self.parameters():
+            #     p.send_(*dest, **kwargs)
```

Comment:
Looks like the code below replaces these lines, so probably good to remove them.